### PR TITLE
feat: scroll back to artworks grid

### PR DESCRIPTION
**Pull request by MounirDhahri**
### Description
chore: scroll back to artworks grid on artworks press.

https://github.com/artsy/eigen/assets/11945712/d03a02d2-a72b-4779-b55b-a68013e813cd

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

#nochangelog

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/routes.ts [hunk 1]
```diff
@@ -140,7 +140,8 @@ function getDomainMap(): Record<string, RouteMatcher[] | null> {
     })),
     addRoute("/artist/:artistID/artworks", "Artist", (params) => ({
       ...params,
-      initialTab: "Insights",
+      initialTab: "Artworks",
+      scrollToArtworksGrid: true,
```
Comment (gkartalis) [inline src/app/routes.ts:144]: 👏 oof why was that insights before? 😅

**Comment by MounirDhahri** [comment]
Merging this to make sure it makes it to the release

**Review by gkartalis — APPROVED** [review]

**Comment by gkartalis** [comment]
> Merging this to make sure it makes it to the release

Let's make sure that @MrSltun is aware to cherry pick

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ✅ **No changelog changes**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0b975ffbe0bf201fcf82d55602ae4c7dd7642e4f
</p>

**Comment by MrSltun** [comment]
Will cherry pick this after the QA today due to not having enough time to build new betas

**Pull request merged**